code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react'
+import { useParams, Link } from 'react-router-dom'
+import { BackButton } from '../BackButton'
+
+import 'pages/movieDetails.css'
+import '../backButton.css'
+
+export const MovieDetails = () => {
+  const { movieId } = useParams()
+  const [movie, setMovie] = useState([])
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=ce155d55b3ce81f17dbaf865fc989515&language=en-US`)
+      .then((res) => res.json())
+      .then((json) => {
+        console.log(json)
+        setMovie(json)
+      })
+  }, [movieId])
+
+  return (
+    <div className='backdrop' style={{ backgroundImage: `url(https://image.tmdb.org/t/p/original/${movie.backdrop_path})` }}>
+      <div className='backdrop-gradient'>
+        <div className='content-container'>
+          <img className='details-poster' src={`https://image.tmdb.org/t/p/w500/${movie.poster_path}`} alt={movie.title} />
```

Comment:
Nice use of the alt!

Reply:
Thank you! Although, I can't take credit. The idea came from a codealong from Damien :)